Camera view with .NET 9 crashes on Android

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [x] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

There is an existing bug for this but it was closed and is now locked for updating: https://github.com/CommunityToolkit/Maui/issues/2377

I have the exact same issue as described in that bug so I suspect it might not have been properly fixed.

My stacktrace:

01-13 13:49:15.312 F/mono-rt ( 5558): [ERROR] FATAL UNHANDLED EXCEPTION: System.MissingMethodException: Method not found: void AndroidX.Camera.Core.Preview.SetSurfaceProvider(AndroidX.Camera.Core.Preview/ISurfaceProvider)
01-13 13:49:15.312 F/mono-rt ( 5558):    at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[<StartUseCase>d__14](<StartUseCase>d__14& stateMachine)
01-13 13:49:15.312 F/mono-rt ( 5558):    at CommunityToolkit.Maui.Core.CameraManager.StartUseCase(CancellationToken token)
01-13 13:49:15.312 F/mono-rt ( 5558):    at CommunityToolkit.Maui.Core.CameraManager.<>c__DisplayClass41_0.<<PlatformConnectCamera>b__0>d.MoveNext() in /_/src/CommunityToolkit.Maui.Camera/CameraManager.android.cs:line 164
01-13 13:49:15.312 F/mono-rt ( 5558): --- End of stack trace from previous location ---
01-13 13:49:15.312 F/mono-rt ( 5558):    at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
01-13 13:49:15.312 F/mono-rt ( 5558):    at Android.App.SyncContext.<>c__DisplayClass2_0.<Post>b__0() in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.App/SyncContext.cs:line 36
01-13 13:49:15.312 F/mono-rt ( 5558):    at Java.Lang.Thread.RunnableImplementor.Run() in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Java.Lang/Thread.cs:line 37
01-13 13:49:15.312 F/mono-rt ( 5558):    at Java.Lang.IRunnableInvoker.n_Run(IntPtr jnienv, IntPtr native__this) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/obj/Release/net9.0/android-35/mcw/Java.Lang.IRunnable.cs:line 59
01-13 13:49:15.312 F/mono-rt ( 5558):    at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PP_V(_JniMarshal_PP_V callback, IntPtr jnienv, IntPtr klazz) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNINativeWrapper.g.cs:line 22







### Expected Behavior

No crash

### Steps To Reproduce

See Current Behavior

### Link to public reproduction project repository

https://github.com/MarianHristov92/MauiCameraViewNet9

### Environment

```markdown
- .NET MAUI CommunityToolkit:10.0.0
- .NET MAUI CommunityToolkit.Maui.Camera: 2.0.0
- OS: ANDROID 15
- .NET MAUI: 9.0.22
```

### Anything else?

_No response_

--- a/src/CommunityToolkit.Maui.Camera/CameraManager.android.cs
+++ b/src/CommunityToolkit.Maui.Camera/CameraManager.android.cs
@@ -172,7 +172,7 @@ partial class CameraManager
 
 	protected async Task StartUseCase(CancellationToken token)
 	{
-		if (resolutionSelector is null)
+		if (resolutionSelector is null || cameraExecutor is null)
 		{
 			return;
 		}
@@ -183,7 +183,7 @@ partial class CameraManager
 		imageCapture?.Dispose();
 
 		cameraPreview = new Preview.Builder().SetResolutionSelector(resolutionSelector).Build();
-		cameraPreview.SetSurfaceProvider(previewView?.SurfaceProvider);
+		cameraPreview.SetSurfaceProvider(cameraExecutor, previewView?.SurfaceProvider);
 
 		imageCapture = new ImageCapture.Builder()
 		.SetCaptureMode(ImageCapture.CaptureModeMaximizeQuality)

--- a/src/CommunityToolkit.Maui.Camera/CommunityToolkit.Maui.Camera.csproj
+++ b/src/CommunityToolkit.Maui.Camera/CommunityToolkit.Maui.Camera.csproj
@@ -51,8 +51,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">
-    <PackageReference Include="Xamarin.AndroidX.Camera.Camera2" Version="1.3.1.1" />
-    <PackageReference Include="Xamarin.AndroidX.Camera.View" Version="1.3.1.1" />
+    <PackageReference Include="Xamarin.AndroidX.Camera.Camera2" Version="1.4.1" />
+    <PackageReference Include="Xamarin.AndroidX.Camera.View" Version="1.4.1" />
 
     <!--  Ensure Linker does not remove required libraries -->
     <None Include="linker.xml" Pack="true" PackagePath="build\$(PackageId).LinkerConfigurationFile.xml" />

